http-host/index.js

```diff
@@ -33,12 +35,17 @@ app.post('/fake_server_admin/calls', ({ body: { method: mockedMethod, url: mocke
     mock = fakeServer.http[mockedMethod]()
       .to(mockedUrl);
   }
-  if (statusCode && statusCode !== 200) {
+  if (statusCode && statusCode >= 400) {
     call = mock
       .willFail(statusCode);
   } else {
+    let finalResponse = mockedResponse;
+    if (finalResponse) {
+      finalResponse = isJson ? JSON.parse(finalResponse) : finalResponse;
```

Comment:
isJson -> responseAsJson

Reply:
I thought about this already, but the same terminology is used in both server and client, changing these would be a breaking change.